native-platform/src/main/java/net/rubygrapefruit/platform/internal/DefaultPosixFiles.java

```diff
@@ -66,8 +66,12 @@ public void setMode(File file, int perms) {
         }
     }
 
-    public int getMode(File file) {
-        PosixFileInfo stat = stat(file);
+    public int getMode(File file) throws NativeException {
+        return getMode(file, false);
+    }
+
+    public int getMode(File file, Boolean linkTarget) throws NativeException {
```

Comment:
:x: why is this `Boolean` instead of `boolean`?

Reply:
Nice catch, that's a typo